lib/classes/PluginManager.js

```diff
@@ -148,7 +151,25 @@ class PluginManager {
     if (pluginsObject.localPath) {
       module.paths.unshift(pluginsObject.localPath);
     }
-    this.loadPlugins(pluginsObject.modules);
+    this.loadPlugins(pluginsObject.modules
+      .filter(name => name !== '@serverless/enterprise-plugin'));
+  }
+
+  loadEnterprisePlugin() {
+    if (config.getGlobalConfig().enterpriseDisabled) {
+      return;
+    }
+    module.paths.unshift(path.join(__dirname, '../../node_modules'));
```

Comment:
I might've asked this before, but this will work on Windows, since `path.join` is able to turn `/` into `\`, correct?

Reply:
oh right.. i tested this when i did it the other way (hacking it into a relative import). I think this'll work, but I'll boot up my window sbox to double check

Reply:
works!